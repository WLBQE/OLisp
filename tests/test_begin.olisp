(print (begin (print 1) (print "hello") 888))
