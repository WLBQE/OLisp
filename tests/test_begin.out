1
hello
888
